src/app/auth/login/page.jsx

```
@@ -134,180 +137,364 @@
+                <h1 className="text-4xl xl:text-5xl font-bold mb-6 leading-tight">
+                  <span className="text-white block mb-2">Continue Your</span>
+                  <span className="bg-gradient-to-r from-[#FF6B00] to-[#FF9A00] bg-clip-text text-transparent inline-block relative">
+                    Ancient Journey
+                    <svg
+                      width="100%"
+                      height="8"
+                      className="absolute -bottom-2 left-0"
+                      viewBox="0 0 400 8"
+                      fill="none"
+                      xmlns="http://www.w3.org/2000/svg"
+                    >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)